sos/report/plugins/maas.py

```diff
@@ -19,16 +19,22 @@ class Maas(Plugin, UbuntuPlugin):
     profiles = ('sysmgmt',)
     packages = ('maas', 'maas-common')
 
-    services = (
+    deb_services = (
         'maas-dhcpd',
         'maas-dhcpd6',
         'maas-http',
         'maas-proxy',
         'maas-rackd',
         'maas-regiond',
-        'maas-syslog'
+        'maas-syslog',
     )
 
+    snap_services = (
+        'snap.maas.supervisor',
+    )
+
+    services = deb_services + snap_services
```

Comment:
I think we can just do one services list and collect all of them every time (not if/then on snap/deb).  Information on non-existent services does not get collected (just tried it out).

Reply:
OK - I've squashed them back together and commented to highlight expectations